### PR TITLE
Unitests and config file

### PR DESCRIPTION
**Pull request by danielpastor97**
## Description
Management of the parameters for the calculation using a config file. Also some updates on the unittests.

## Notable points that this PR has either accomplished or will accomplish.
  - [x] Implementation of a config file. I used the built-in ``configparser`` library and created a default.ini file that includes the parameters that the software uses by default but that can be updated using the defined options in the command-line version.
  - [x] Partial updating of unittests.

## Status
- [x] Ready to go

### CODE REVIEW

File: ufcc/default.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[Parameters]
+cutoff = 7
+lipid_types = POPC, DPPC, DOPC, CHOL, CHL1, POPS, POPE
+backend = serial
+n_jobs = -1
+
+[Report]
+build-report = True
```
Comment (bisejdiu) [inline ufcc/default.ini:8]: By default, we should not output a report. The default outputs should be either the CSV or the dashboard (or both).

File: ufcc/default.ini [hunk 2]
```diff
@@ -0,0 +1,9 @@
+[Parameters]
+cutoff = 7
```
Comment (bisejdiu) [inline ufcc/default.ini:2]: Add 1 or 2 lines of documentation for each parameter (what it is, how it is used).

File: ufcc/default.ini [hunk 3]
```diff
@@ -0,0 +1,9 @@
+[Parameters]
```
Comment (bisejdiu) [inline ufcc/default.ini:1]: Add the following parameters (with the provided comments): 

```python
# The number of residues to show when rendering Lipid Contact Durations
# reasonable values should be in the range 5-20
residues_to_show = 15

# Eliminate spurious contacts (i.e. short-lived contacts). This also speeds up calculations a little bit. 
# Values are the number of frames. Contacts that fall below are discarded. 
intervals_to_filter_out = 10

# Tolerance for not satisfying the cutoff. When measuring contact durations, we can allow a lipid to 
# escape the cutoff region by this number of frames and still count it in contact. 
tolerance = 6
```

**Review by bisejdiu — CHANGES_REQUESTED** [review]

**Comment by danielpastor97** [comment]
Hey @bisejdiu , everything is solved, notice that the comments in the config file are with `;` instead of `#`. Please let me know if any other changes are required in this regard.  Cheers!

**Pull request merged**